### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by LukaJCB**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,14 +6,14 @@ object Dependencies {
 
   //noinspection TypeAnnotation
   object Versions {
-    val scalaCheck = "1.13.4"
-    val scalaTest = "3.0.0"
+    val scalaCheck = "1.13.5"
+    val scalaTest = "3.0.5"
     val discipline = "0.8"
     val macroParadise = "2.1.1"
     val kindProjector = "0.9.5"
```
Comment (barambani) [inline project/Dependencies.scala]: would it be doable to update this to 0.9.6 as part of this ?

**Comment by rossabaker** [comment]
:+1:

**Comment by snadorp** [comment]
@kailuowang Can you release a version with this?

**Comment by kailuowang** [comment]
@snadorp will do soon

**Comment by snadorp** [comment]
Awesome thanks.

**Pull request merged**